garbagetruck/garbagetruck.go

```diff
@@ -0,0 +1,59 @@
+package garbagetruck
+
+import (
+	"github.com/ian-kent/go-log/logger"
```

Comment:
this is MIT licensed external component. Where do we represent this information when we build webpa ?